test/views/ViewSpecBase.scala

```diff
@@ -66,6 +66,28 @@ trait ViewSpecBase extends SpecBase {
 
   }
 
+  def assertPageTitleWithSectionSubheading(doc: Document,
+                                           expectedMessageKey: String,
+                                           captionParam: String,
+                                           args: Any*): Assertion = {
```

Comment:
By looking at the usage of this method captionParam and args are both redundant and can be removed
This
${messages(s"$expectedMessageKey.caption", captionParam) and just be
${messages(s"$expectedMessageKey.caption")

Similarly
     val expectedHeading = messages(s"$expectedMessageKey.heading", args:_*)
becomes
     val expectedHeading = messages(s"$expectedMessageKey.heading")

Reply:
I didn't change this one as it's in the test and I'd already applied the other changes to all the projects